HealthEndpoints.md

```diff
@@ -0,0 +1,43 @@
+# The health endpoints
+These are technical endpoints of no use to somebody looking for facts.
+  
+There are 2 health endpoints:
+```
+/health/ready
+/health/alive
+```
+
+Their usefullness depend on how Pxwebapi is installed. 
+If you have a single pxwebapi instance and install it as a package,
+ you will not find them very usefull. 
+
+Their intended use is if you run Pxwebapi as Docker in a kubernetes cluster or 
+have more than one server behind a loadbalancer.
```

Comment:
I think maintenance mode is not for Kubernetes deployments,  but it's useful when you have dedicated serveres behind a loadbalancer. Like we do, on-prem :-)

Reply:
I "googled with AI" They suggest PUT with true false in body ?  
The get would be for "internal use" to see why health/ready is not 200 ?